.github/workflows/build-notebooks-pr.yaml

```
@@ -33,7 +33,8 @@ jobs:
           git fetch --no-tags origin '+refs/heads/${{ github.event.pull_request.base.ref }}:refs/remotes/origin/${{ github.event.pull_request.base.ref }}'
           python3 ci/cached-builds/gen_gha_matrix_jobs.py \
             --from-ref 'origin/${{ github.event.pull_request.base.ref }}' \
-            --to-ref '${{ github.event.pull_request.head.ref }}'
+            --to-ref '${{ github.event.pull_request.head.ref }}' \
+            --leave-out-rhel '${{ github.event_name == 'pull_request' }}'
```

[based on the failure](https://github.com/red-hat-data-services/notebooks/actions/runs/13861154180/job/38789730801?pr=514), maybe this should be:
```suggestion
            --leave-out-rhel='${{ github.event_name == 'pull_request' }}'
```